components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -10,291 +10,789 @@ toc = true
+```
 
-In the IAM v2 beta, policies must be deleted from the command line.
+### Creating a User
 
-See [Listing Policies]({{< relref "iam-v2-api-reference.md#listing-policies" >}}) to fetch a list of all your policies and locating the relevant policy ID.
-Use the policy ID as the final path component to delete a policy:
+Create a user by composing JSON with all the necessary user properties (see [Example User]({{< relref "iam-v2-api-reference.md#example-user" >}})).
+
+Assuming you store it in the file "user.json", pass it to `curl` to create the user:
 
 ```bash
-curl -s -H "api-token: $TOKEN" -X DELETE https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/policies/{policy-id}?pretty
+curl -sSH "api-token: $TOKEN" -d @user.json -X POST \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/users?pretty
 ```
 
-In the browser: Navigate to the *Policies* page under the **Settings** tab., and then find the policy you wish to delete.
-Use the menu at the end of the policy row and select the **Delete Policy** option to delete the policy.
+### Updating a User
 
-## Roles
+```bash
+curl -sSH "api-token: $TOKEN" -d @user.json -X PUT \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/users/{user-id}?pretty
+```
 
-A role encapsulates a list of actions.
-IAM v2 provides several default Chef-managed roles, which are essential to the operation of Chef Automate and cannot be altered.
-Roles you create are Custom roles.
+- Supply all of a user's properties, not just the ones you wish to update.
+  Properties that you do not include are reset to empty values.
+- The user ID is immutable; it can only be set at creation time.
 
-Chef-managed Role Name | ID| Actions
------------------------|-----|--------
-Owner       | owner       | `*`
-Viewer      | viewer      | `infra:*:get`, `infra:*:list`, `compliance:*:get`, `compliance:*:list`, `system:*:get`, `system:*:list`, `event:*:get`, `event:*:list`, `ingest:*:get`, `ingest:*:list`, `applications:*:list`, `applications:*:get`
-Editor      | editor      | `infra:*`, `compliance:*`, `system:*`, `event:*`, `ingest:*`, `secrets:*`, `telemetry:*`, `applications:*`
-Ingest      | ingest      | `infra:ingest:*`, `compliance:profiles:get`, `compliance:profiles:list`
+### Deleting a User
 
-### Listing Roles
+```bash
+curl -sSH "api-token: $TOKEN" -X DELETE \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/users/{user-id}?pretty
+```
+
+Deleting a user is permanent and cannot be undone.
+
+### List Teams for a User
 
 ```bash
-curl -s -H "api-token: $TOKEN" https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/roles?pretty
+curl -sSH "api-token: $TOKEN" \
+https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/users/{membership-id}/teams?pretty
 ```
 
-The output details each role name, role type, and actions it contains.
+The output lists all teams that contain the specified user.
+Note, it uses the `membership_id` property of the user instead of the user's `id`.
 
```

Comment:
Need to add missing endpoint here. (Remove the spaces between the backticks on line 4)
```
### Updating Your Own User

```bash
curl -sSH "api-token: $TOKEN" -d @user.json -X PUT \
https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/self/{user-id}?pretty
` ` `

- Supply all of your user properties, not just the ones you wish to update.
  Properties that you do not include are reset to empty values.
- Your user ID is immutable; it can only be set at creation time.
```